web/src/app/main/URLErrorDialog.js

```diff
@@ -1,58 +1,37 @@
 import React from 'react'
-import { urlParamSelector } from '../selectors'
-import { connect } from 'react-redux'
-import { resetURLParams } from '../actions'
 import { Dialog, DialogContent, DialogActions } from '@material-ui/core'
 import DialogTitleWrapper from '../dialogs/components/DialogTitleWrapper'
 import DialogContentError from '../dialogs/components/DialogContentError'
 import LoadingButton from '../loading/components/LoadingButton'
 
-@connect(
-  (state) => ({
-    errorMessage: urlParamSelector(state)('errorMessage'),
-    errorTitle: urlParamSelector(state)('errorTitle'),
-  }),
-  (dispatch) => ({
-    resetError: () => dispatch(resetURLParams('errorMessage', 'errorTitle')),
-  }),
-)
-export default class URLErrorDialog extends React.Component {
-  handleDialogClose = () => {
-    this.props.resetError()
-  }
+export default function URLErrorDialog({ errorMessage, errorTitle, onClose }) {
```

Comment:
Thank you for the useful feedback. But I wonder if we sill need the 
`handleDialogClose = () => {
    this.props.resetError()
  }`
function anymore because it's used nowhere in the body component. And the same with the dispatch function `resetError` whis is being called inside that function